package.json

```diff
@@ -15,6 +15,7 @@
     "webpack-dev-server": "webpack-dev-server",
     "protractor": "protractor",
     "build": "npm run build:dev",
+    "clean:dist": "rm -rf dist",
```

Comment:
We may only have need to run this on *nix systems now, but it would generally be better to use some platform-independent build tool like [clean-webpack-plugin](https://github.com/johnagan/clean-webpack-plugin) for this task.